fli-iam/shanoir-ng
# shanoir-issue#1943: send dicom data to pre-fill center and acq equipment forms during imports

Fix issue #1943

I haven't been able to test the import of a CT Dicom that contains a device serial number (I don't have the data).

How to test:
Import a DICOM
Select a zip file of CT dataset
Select the CT series
On step 3 of import, add a center
The 'create center' form should be pre-filled with the DICOM's data (InstitutionName and InstitutionAddress) if not empty
Once the center is selected, create an acquisition equipment, it should also be pre-fill with the serial number (Device Serial Number in DICOM metadata)
